Sources/Sentry/include/SentryProfilingLogging.hpp

```diff
@@ -19,12 +21,22 @@ namespace profiling {
 } // namespace profiling
 } // namespace sentry
 
-#define SENTRY_PROF_LOG_DEBUG(...)                                                                 \
-    sentry::profiling::log(sentry::profiling::LogLevel::Debug, __VA_ARGS__)
-#define SENTRY_PROF_LOG_WARN(...)                                                                  \
-    sentry::profiling::log(sentry::profiling::LogLevel::Warning, __VA_ARGS__)
-#define SENTRY_PROF_LOG_ERROR(...)                                                                 \
-    sentry::profiling::log(sentry::profiling::LogLevel::Error, __VA_ARGS__)
+#    define SENTRY_PROF_LOG_DEBUG(...)                                                             \
+        sentry::profiling::log(sentry::profiling::LogLevel::Debug, __VA_ARGS__)
+#    define SENTRY_PROF_LOG_WARN(...)                                                              \
+        sentry::profiling::log(sentry::profiling::LogLevel::Warning, __VA_ARGS__)
+#    define SENTRY_PROF_LOG_ERROR(...)                                                             \
+        sentry::profiling::log(sentry::profiling::LogLevel::Error, __VA_ARGS__)
+
+#else
+
+// don't do anything with these in production until we can get a logging solution in place that
```

Comment:
add a brief explanation and a link to my comment on the issue (https://github.com/getsentry/sentry-cocoa/issues/3336#issuecomment-1802892052) so we have a reference for why using NSLog here is bad